.github/workflows/pull-evaluation-tests.yaml

```diff
@@ -200,6 +200,8 @@ jobs:
           echo "AICORE_AUTH_URL=$(yq eval '.stringData.auth_url' aicore-secret.yaml)" >> $GITHUB_OUTPUT
           echo "AICORE_BASE_URL=$(yq eval '.stringData.base_url' aicore-secret.yaml)" >> $GITHUB_OUTPUT
           echo "AICORE_RESOURCE_GROUP=$(yq eval '.stringData.resource_group' aicore-secret.yaml)" >> $GITHUB_OUTPUT
+          echo "AICORE_CONFIGURATION_ID_GPT4_MINI=$(yq eval '.stringData.configuration_id_gpt4_mini' aicore-secret.yaml)" >> $GITHUB_OUTPUT
+          echo "AICORE_DEPLOYMENT_ID_GPT4_MINI=$(yq eval '.stringData.deployment_id_gpt_mini4' aicore-secret.yaml)" >> $GITHUB_OUTPUT
           echo "AICORE_CONFIGURATION_ID_GPT4=$(yq eval '.stringData.configuration_id_gpt4' aicore-secret.yaml)" >> $GITHUB_OUTPUT
```

Comment:
can these be removed if mini is used now?

Reply:
Yes.

Reply:
Done